tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/PetResource.java

```diff
@@ -191,33 +194,37 @@ public Response addPet(Pet pet) {
     })
     @Operation(summary = "Update an existing pet", description = "Update an existing pet with the given new attributes")
     public Response updatePet(
-            @Parameter(name = "petAttribute", description = "Attribute to update existing pet record", required = true,
-                       schema = @Schema(implementation = Pet.class)) Pet pet) {
+            @RequestBody(name = "petAttribute", description = "Attribute to update existing pet record",
+                         required = true,
+                         content = @Content(schema = @Schema(implementation = Pet.class))) Pet pet) {
```

Comment:
I don't think `name` should be set when using `@RequestBody` on a parameter like this.

Also, although the original annotation is definitely wrong, replacing it with the correct annotation is a little close to expanding the test. As nothing appears to assert this, I'd be tempted to just remove the offending annotation.

Reply:
Removed the `name`

Reply:
> replacing it with the correct annotation is a little close to expanding the test. As nothing appears to assert this, I'd be tempted to just remove the offending annotation.

Since there aren't any existing assertions on the body for this operation, I think we should keep it for now. Although it's likely to alter the output of an implementation, it's more in the spirit of this PR. It would probably be a good exercise to go through a second time and prune out methods/annotations that are not actually part of the tests.

Let me know if that sounds OK to you.

Reply:
ok